OptimizelySDK/Config/HttpProjectConfigManager.cs

```diff
@@ -182,6 +183,20 @@ public Builder WithErrorHandler(IErrorHandler errorHandler)
                 return this;
             }
 
+            public Builder WithAutoUpdate(bool autoUpdate)
+            {
+                AutoUpdate = autoUpdate;
+
+                return this;
+            }
+
+            public Builder WithStartByDefault(bool startByDefault=true)
```

Comment:
@mjc1283 @mikeng13 You are right but I added `Start` because there are chances to lost first `Notification`. Even if you `Start` by default `HttpProjectConfigManager`, it emits event before subscribing and initializing `Optimizely` instance.
To overcome this issue, I think `NotificationCenter` should be passed to `HttpProjectConfigManager`, like implemented in Java. If you agree I can make this change?

Reply:
That change sounds good to me. Thanks.